cmd/esc/cli/env_diff.go

```diff
@@ -24,15 +23,18 @@ func newEnvDiffCmd(env *envCommand) *cobra.Command {
 	diff := &envGetCommand{env: env}
 
 	cmd := &cobra.Command{
-		Use:   "diff [<org-name>/]<environment-name>[@<version>] [@<version>]",
+		Use:   "diff [<org-name>/]<environment-name>[@<version>] [[[org-name/]<environment-name>]@<version>]",
 		Args:  cobra.RangeArgs(1, 2),
 		Short: "Show changes between versions.",
 		Long: "Show changes between versions\n" +
 			"\n" +
-			"This command displays the changes between two versions of an environment.\n" +
-			"The first argument is the environment and base version for the diff and\n" +
-			"the second argument is the comparison version. If the second argument is\n" +
-			"omitted, the 'latest' tag is used.",
+			"This command displays the changes between two environments or two versions\n" +
+			"of a single environment.\n" +
+			"\n" +
+			"The first argument is the base environment for the diff and the second argument\n" +
+			"is the comparison environment. If the environment name portion of the second\n" +
+			"argument, the name of the base environment is used. If the version portion of\n" +
```

Comment:
```suggestion
			"is the comparison environment. If the environment name portion of the second\n" +
			"argument is omitted, the name of the base environment is used. If the version portion of\n" +
```

I think you omitted some words :)